mitiq/pec/types/types.py

```diff
@@ -356,23 +178,6 @@ def add(self, *basis_elements: Sequence["NoisyOperation"]) -> None:
                 )
             self._basis_elements.add(noisy_op)
 
-    def extend_to(self, qubits: Sequence[List[cirq.Qid]]) -> None:
```

Comment:
Same question as in `NoisyOperation`, although here it's less important since I believe the plan is to remove `NoisyBasis` eventually.

Reply:
Exactly, `NoisyBasis` will be removed anyway in the next PR.